webapp/components/post_view/reaction/reaction.jsx

```diff
@@ -77,6 +79,10 @@ export default class Reaction extends React.PureComponent {
         this.removeReaction = this.removeReaction.bind(this);
     }
 
+    componentDidMount() {
+        postListScrollChange();
```

Comment:
I think this should be done in the ReactionList when the number of reactions that it's rendering changes. That way, it'll only fire once for a post when its first loaded instead of once for each reaction on that post